it/controllers/benefits/reimbursed/NonTaxableCostsBenefitsAmountControllerISpec.scala

```diff
@@ -497,7 +489,7 @@ class NonTaxableCostsBenefitsAmountControllerISpec extends IntegrationTest with
 
     }
 
-    "update cya when a user submits a valid form and doesn't have prior benefits" which {
+    "update cya when a user submits a valid form and doesn't have prior benefits, redirects to the taxable costs page" which {
```

Comment:
"redirect to taxable expenses yes/no question page" can be changed to "taxable costs page"

Reply:
updated